app/main/recipe_import.py

```diff
@@ -0,0 +1,73 @@
+import requests
+from .recipe_parser import  PicoBrewRecipeImport,  ZymaticRecipeImport,ZSeriesRecipeImport
+from .config import MachineType
+from flask import current_app
+SYNC_URI_TMPLS = {
+    MachineType.PICOBREW: 'http://137.117.17.70/API/pico/getRecipe?uid={}&rfid={}&ibu=-1&abv=-1.0',
+    MachineType.ZYMATIC: 'http://137.117.17.70/API/SyncUSer?user={}&machine={}'
+}
+
+
+
+class ImportException(Exception):
+    pass
+
+# id should be the rfid of a picopack, or a user GUID
+def import_recipes(mach_uid: str, account_id: str, mach_type: MachineType):
+    if mach_type == MachineType.ZYMATIC or mach_type == MachineType.PICOBREW:
+        return import_recipes_classic(mach_uid, account_id, mach_type)
+    elif mach_type == MachineType.ZSERIES:
+        return import_recipes_z(mach_uid)
+
+
+def import_recipes_classic(mach_uid, account_id, mach_type):
```

Comment:
Remove/Update/test existing code if you're adding helpers for Pico/Zymatic imports here